### ISSUE TITLE
NPE in slack_websocket_cb

### ISSUE THREAD

**Issue by Quentin-M** [rawdigits/wee-slack#143]
Reporting an error:

```
python: error in function "slack_websocket_cb"
python: wrong arguments for function "buffer_set" (script: slack_extension)
python: stdout/stderr: TypeError: must be string, not None
python: error in function "slack_websocket_cb"
```

I will fix it if I find the time.

**Comment by tfheen** [comment]
]] Quentin Machu 

> python: error in function "slack_websocket_cb"
> python: wrong arguments for function "buffer_set" (script: slack_extension)
> python: stdout/stderr: TypeError: must be string, not None
> python: error in function "slack_websocket_cb"

There should be a traceback too. Can you provide that?

-- 
Tollef Fog Heen
UNIX is user friendly, it's just picky about who its friends are

**Comment by Quentin-M** [comment]
Nah, it did not appear in my buffer. I just enabled slack-debug. I will see if it appears again.

**Comment by tfheen** [comment]
Hmm, are you running an up-to-date version? I don't see any buffer_set in the websocket callback function at all.

**Comment by tfheen** [comment]
Hmm, I'm wondering if it's a duplicate of #138

**Comment by etu** [comment]
This started happening when I did `/buffer merge 1` on the slack status buffers:

```
23:41:25                        weechat =!= | python: wrong arguments for function "nicklist_search_nick" (script: slack_extension)
23:41:25                        weechat =!= | python: wrong arguments for function "nicklist_remove_nick" (script: slack_extension)
23:41:25                        weechat     | python: stdout/stderr: Traceback (most recent call last):
23:41:25                        weechat     | python: stdout/stderr:   File "/home/etu/.weechat/python/autoload/wee_slack.py", line 1282, in slack_websocket_cb
23:41:25                        weechat     | python: stdout/stderr:     proc[function_name](message_json)
23:41:25                        weechat     | python: stdout/stderr:   File "/home/etu/.weechat/python/autoload/wee_slack.py", line 1338, in process_presence_change
23:41:25                        weechat     | python: stdout/stderr:     server.users.find(identifier).set_active()
23:41:25                        weechat     | python: stdout/stderr:   File "/home/etu/.weechat/python/autoload/wee_slack.py", line 801, in set_active
23:41:25                        weechat     | python: stdout/stderr:     channel.update_nicklist(self.identifier)
23:41:25                        weechat     | python: stdout/stderr:   File "/home/etu/.weechat/python/autoload/wee_slack.py", line 447, in update_nicklist
23:41:25                        weechat     | python: stdout/stderr:     w.nicklist_add_nick(self.channel_buffer, here, user.name, user.color_name, "", "", 1)
23:41:25                        weechat     | python: stdout/stderr: UnboundLocalError: local variable 'here' referenced before assignment
23:41:25                        weechat =!= | python: error in function "slack_websocket_cb"
```

And it keeps on repeating. Had them showing up at `23:31:22`, `23:35:14`, `23:37:39`, `23:38:09`, `23:41:25`, `23:44:04` by just sitting in buffer 1 with statusbuffers and doing nothing. No activity in any other buffer except one IRC buffer.

**Comment by fuzzy76** [comment]
I'm seeing this too. This constant flooding is making the plugin unusable for me. :(

**Comment by rawdigits** [comment]
Fixed by #178
